fulp_modules/minebot_improvements/minebot_additions.dm

```diff
@@ -0,0 +1,101 @@
+// This is the file for the small improvements I made to Minebots.
+// It's meant to be fully modular and to be a good example of how to
+// implement such modularity. Enjoy.
+
+// Starting off by adding some stuff to the initial definition of minebots
+/mob/living/simple_animal/hostile/mining_drone
+	..()
```

Comment:
There is no need to add "..()" in there, it already includes all the variables from /tg/ file.